placementPortal/urls.py

```diff
@@ -41,8 +41,9 @@
         schema_view.with_ui("swagger", cache_timeout=0),
         name="schema-swagger-ui",
     ),
+    path("", include("placementApp.urls")),
     path("api/auth/", include("djoser.urls.authtoken")),
     path("api/auth/", include("djoser.urls")),
     path("api/auth/", include("djoser.urls")),
-    path("", include("placementApp.urls")),
```

Comment:
Remove the extra path.
Also remove the endpoint for type added earlier along with its views

Reply:
done